.github/workflows/continuous-integration.yml

```diff
@@ -52,6 +52,8 @@ jobs:
         run: yarn test
         env:
           NODE_ENV: production
+      - name: 'Assert no new snapshots (run `yarn test --ci` if this fails)'
+        run: git diff --exit-code
```

Comment:
@kyubisation I don't know much about github actions :/
Confirm if it's correct pls

Reply:
Looks correct to me 👍 
(This basically runs "git diff --exit-code" and if the exit code is != 0, it errors. In the logs this will be shown as a step in the test job, with the given name.)